### PR TITLE
Show movies

### PR DESCRIPTION
**Pull request by kaylalarson1990**
@emilydittmer @Nathan-Froeh got the movie data to show up on cards and pull in mapStateToProps in our movieContainer. Let me know if you have any questions!

### CODE REVIEW

File: src/Containers/MovieContainer/MovieContainer.jsx [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-      <MovieCard/>
-      <MovieCard/>
-      <MovieCard/>
-    </section>
-  )
-}
-
-export default MovieContainer;
\ No newline at end of file
+export default connect(mapStateToProps)(MovieContainer);
```
Comment (emilydittmer) [inline src/Containers/MovieContainer/MovieContainer.jsx]: Thanks for adding this in! It looks great. Now that we have this, we can look at conditionally rendering additional information on the click of the movie card.

File: src/Containers/MovieContainer/MovieContainer.jsx [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+  const displayMovies = props.movies.movies.map(movie => (
+    <MovieCard {...movie} key={movie.id} title={props.title} />
+  ));
+  return <section>{displayMovies}</section>;
+};
 
-const MovieContainer = () => {
+const mapStateToProps = state => ({
+  movies: setMovies(state.movies)
+});
```
Comment (emilydittmer) [inline src/Containers/MovieContainer/MovieContainer.jsx]: I think once we get to the conditional render, we'll add the mapDispatchToProps to add the onClick function, and adding the favorite icon.

File: src/Containers/MovieCard/MovieCard.jsx [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
-      <p className="year">2019</p>
-    </article>
-  );
-};
+export class MovieCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      favorites: null,
+      isFav: false,
```
Comment (emilydittmer) [inline src/Containers/MovieCard/MovieCard.jsx]: This looks good. However, will this just be in state of the card, or will this also need to be in the store?

File: src/Containers/MovieCard/MovieCard.jsx [hunk 4]
```diff
@@ -1,17 +1,26 @@
-import React from "react";
+import React, { Component } from "react";
```
Comment (emilydittmer) [inline src/Containers/MovieCard/MovieCard.jsx]: Also, will this be a container or component? We may have to move this to the component file after we look into this further.

File: src/Components/App/App.jsx [hunk 5]
```diff
@@ -4,32 +4,40 @@ import Aside from '../Aside/Aside.jsx'
...
-    super();
-    this.state = {
-      movies: []
-    };
-  }
+
   componentDidMount() {
     nowPlaying()
-      .then(movies => this.setState({ movies: movies}))
+      .then(movies => this.props.setMovies(movies))
```
Comment (emilydittmer) [inline src/Components/App/App.jsx]: Awesome! Thanks for adding this.

File: src/Containers/MovieCard/MovieCard.jsx (hunk 4 above)
Comment (kaylalarson1990) [reply src/Containers/MovieCard/MovieCard.jsx]: I'm going to delete this as well - we can ask travis to be sure though but I think it stays as component

**Pull request closed without merging**